<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tapahtumat</class>
 <widget class="QDialog" name="tapahtumat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1703</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 255, 237);</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>230</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/poorbanklogo.png);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kirjautunutLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: transparent;
	background: none;
	color: rgb(0, 61, 0);
	border: none;
	selection-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Olet nyt kirjautunut sisään,</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kayttajaNimi">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background: transparent;
	border: none;
	color: rgb(0, 61, 0);
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tapahtumaKentta">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>821</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: white;
    border: 2px solid green;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seuraavaNappi">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>590</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/buttontausta.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}
QPushButton:pressed{
	border-image: url(:/buttonpainettu.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>seuraavaNappi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takaisinNappi">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>590</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/buttontausta.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}
QPushButton:pressed{
	border-image: url(:/buttonpainettu.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>takaisinNappi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kirjauduUlosNappi">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>10</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>28</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/buttontausta.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}
QPushButton:pressed{
	border-image: url(:/buttonpainettu.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nappiTakaisin">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>630</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/buttontausta.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}
QPushButton:pressed{
	border-image: url(:/buttonpainettu.png);
	background-color: transparent;
	color: rgb(255, 255, 255);
	background: none;
	border: none;
	background-repeat: none;
	selection-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Palaa takaisin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
